GREEDY/Services/ReceiptService.cs

```diff
@@ -32,6 +33,7 @@ public Receipt ProcessReceiptImage(Bitmap image)
                 image = _imageFormating.FormatImage(image);
                 var receipt = _receiptCreating.FullReceiptCreating(image);
                 receipt.ItemsList = _dataConverter.ReceiptToItemList(receipt);
+                //receipt.ItemsList = AutoCorrect.ItemsAutoCorrect(receipt.ItemsList);
```

Comment:
Again, write above why this is still left here (can even be in a TODO statement for the future)